src/main/resources/application.yml

```diff
@@ -10,4 +10,8 @@ openapi:
   version: '@project.version@'
 spring:
   application:
-    name: template
+    name: chat
+  datasource:
+    driver-class-name: org.mariadb.jdbc.Driver
```

Comment:
I don't think this is necessary, but I know that we are using it in other services. 
(could be removed a.f.a.i.k)